stats/stats-proto/proto/stats.proto

```diff
@@ -6,36 +6,25 @@ option go_package = "github.com/blockscout/blockscout-rs/stats";
 
 service StatsService {
   rpc GetCounters(CountersRequest) returns (Counters);
-  rpc GetNewBlocks(ChartRequest) returns (ChartInt);
+  rpc GetLine(ChartRequest) returns (Chart);
```

Comment:
What is the reason why this endpoint is named `GetLine` and not `GetChart`. I just cannot get the idea

Reply:
because in future we might get different charts, like pie charts, error bars or something. probably should also change structure name too

may be we could change the name to "points", because some other charts could still have the same structure (bar charts, etc)

Reply:
Get it. I suggest to change http endpoint to be `charts/line` in that case. And maybe to name proto rpc method as `GetLineChart`. It may be easier to get the idea why "line" word is present in that case just looking on the proto and swagger definitions

Reply:
Also, I suggest to name request structures to be `GetCountersRequest` and `GetLineChartRequest` correspondingly. "Get" prefix allows us to add some other types requests for those resources if would be required